jenkins-integration/jenkins-jobs/common/scripts/jenkins/pipeline.groovy

```diff
@@ -175,7 +181,7 @@ def step020_install_server(SKIP_SERVER_INSTALL, script_dir, server_era) {
             withEnv(["PUPPET_SERVER_SERVICE_NAME=${server_era["service_name"]}",
                      "PUPPET_SERVER_TK_AUTH=${server_era["tk_auth"]}",
                      "PACKAGE_BUILD_VERSION=${server_era["version"]}",
-                     "PUPPET_AGENT_VERSION=${server_era["agent_version"]}"]) {
+                     "PUPPET_AGENT_VERSION=${agent_version}"]) {
```

Comment:
I'm not sure I follow - `agent_version` is set on 386 to the return value of `get_agent_version(job["agent_version"])` which I thought reads the value in from the job file? Does referencing `agent_version` here not work that way?